grpc/core/src/main/java/io/helidon/grpc/core/model/ClassAndType.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.core.model;
+
+import java.lang.reflect.Type;
+
+/**
+ * A holder for a {@link Class} and {@link Type}.
+ */
+public class ClassAndType {
```

Comment:
Does this really belong here or could it be moved to Helidon Common, for example?

Seems too generic for gRPC module...